lib/NfcoreTemplate.groovy

```diff
@@ -222,6 +223,21 @@ class NfcoreTemplate {
         }
     }
 
+    //
+    // Dump pipeline parameters in a json file
+    //
+    public static void dump_parameters(workflow, params) {
```

Comment:
there is an issue with the dump parameters file writing on aws. It is already fixed in the template on dev, but might be worth to include here already